.travis.yml

```diff
@@ -13,7 +13,7 @@ addons:
 
 install:
     - sudo apt-get update -qq
-    - sudo apt-get install libtommath-dev
+    - sudo apt-get install libtommath-dev libgmp-dev valgrind
```

Comment:
This is what was needed in ecc branch (latest ltm + tfm)
https://github.com/libtom/libtomcrypt/blob/pr/ecc-asn1-part/.travis.yml#L22

Reply:
yeah, I'd like to stay on standard-built packages as long as possible as the packages that are installed in the ecc branch are built by me ;)